src/main/java/frc/robot/RobotContainer.java

```diff
@@ -113,21 +137,37 @@ public void configureObjects() {
     backLeftWheel.getTurningEncoder().setInverted(true);
     backRightWheel.getTurningEncoder().setInverted(true);
 
+    //Configuring shooter motors
+    leftShooter.setInverted(true);
+    feedMotor.setInverted(true);
+
     coastDrive();
   }
 
   private void configureBindings() {
+    //Constructs input devices
     Joystick leftJoystick = new Joystick(0);
     Joystick rightJoystick = new Joystick(1);
     Joystick altJoystick = new Joystick(2);
 
-    JoystickButton turtleButton = new JoystickButton(rightJoystick, 1);
+    JoystickButton turtleButton = new JoystickButton(rightJoystick, 3);
 
     JoystickButton fieldCentricButton = new JoystickButton(rightJoystick, 2);
```

Comment:
Small thing, but I would think about grouping these button declarations by subsystem or something, instead of giving each it's own code block, simply for better readability.

Reply:
Yeah I was doing something similar but it wasn't full comprehensive, I'll finish it